### PR TITLE
v.cluster: Fix unused value assignment

### PR DESCRIPTION
**Pull request by ShubhamDesai**
This pull request fixes the issue identified by coverity scan (CID: 1270389)

The variable eps gets overwritten so initially assigned value is commented.

### CODE REVIEW

**Comment by lbartoletti** [comment]
We're unsure about the intention behind including both 90% and 99% confidence intervals. Was there originally a plan to have an option between 90% and 99%?
Your suggestion to use comment the line is correct, but if we don't need the 90% interval, it might be better to remove that line entirely.
Also, there's a similar case [at line 289](https://github.com/OSGeo/grass/blob/8c4e378e950a185958614a7999ab481a1f834459/vector/v.cluster/main.c#L289).

**Comment by wenzeslaus** [comment]
> Was there originally a plan to have an option between 90% and 99%?

These would be good things to catch when revising the code. Perhaps all is needed is to create an issue for a (possibly?) straightforward addition (good for beginners). I'm tagging original author @metzm and @neteler who contributed to the documentation quite a bit.

File: vector/v.cluster/main.c [hunk 1]
```diff
@@ -493,7 +493,7 @@ int main(int argc, char *argv[])
 
             mean = sum / n;
             sd = sqrt(sumsq / n - mean * mean);
-            eps = mean + 1.644854 * sd; /* 90% CI */
+            // eps = mean + 1.644854 * sd; /* 90% CI */
```
Comment (neteler) [inline vector/v.cluster/main.c:496]: There is another duplicate `eps` in line 290.

Somehow the code is unexpected, can @metzm enlighten us?

**Comment by petrasovaa** [comment]
I would remove the duplicate eps lines and move on.

**Comment by wenzeslaus** [comment]
> I would remove the duplicate eps lines and move on.

I agree. The meaning of the constant literal is documented and the possibility of switch between 99% and 90% is obvious, so there is no added value in keeping that line or a comment in the code.

File: vector/v.cluster/main.c (hunk 1 above)
Comment (ShubhamDesai) [reply vector/v.cluster/main.c]: Done removed another line as well

**Comment by ShubhamDesai** [comment]
> > I would remove the duplicate eps lines and move on.
> 
> I agree. The meaning of the constant literal is documented and the possibility of switch between 99% and 90% is obvious, so there is no added value in keeping that line or a comment in the code.

Done removed the lines.
Also in the documentation https://grass.osgeo.org/grass84/manuals/v.cluster.html
it mentions "If the maximum distance is not specified with the distance option, the maximum distance is estimated from the observed distances to the neighbors using the upper 99% confidence interval."

so i guess it refers to the 99% confidence interval

**Review by wenzeslaus — APPROVED** [review]
Good. Thanks for checking the documentation.

**Pull request merged**